src/uiTest/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala

```diff
@@ -227,7 +227,9 @@ class UITestSuite extends BaseGitRepositoryBackedIntegrationTestSuite(SETUP_WITH
     val pid = intelliJ.probe.pid()
     intelliJ.probe.screenshot("exception")
     val threadStackTrace: String = Process("jstack " + pid) !!
-    val file: File = new File("/tmp/ide-probe/screenshots/uiTest/exception/thread_dump_" + pid + ".txt")
+    val artifactDirectory = "/tmp/ide-probe/screenshots/uiTest/exception"
+    Files.createDirectories(Paths.get(artifactDirectory))
```

Comment:
some protection against the fact that the screens have not been made and thus directory will not be created. Caught in here:
<img width="965" alt="Screenshot 2022-06-15 at 14 36 26" src="https://user-images.githubusercontent.com/33667003/173828314-89c9a487-77a2-47ef-85a6-b2d57b419a5c.png">